src/entity/BlogTag.ts

```diff
@@ -0,0 +1,20 @@
+import {Entity, PrimaryGeneratedColumn, Column, ManyToOne} from "typeorm";
+
+import {Blog} from "./Blog"; 
+import {Tag} from "./Tag"; 
+
+@Entity()
+export class BlogTag {
+  
+  @PrimaryGeneratedColumn('increment')
+  id: number;
+  
+  @ManyToOne(() => Blog, (blog) => blog.likes)
```

Comment:
what is difference between likes and tags 
tags are like #softwareengineering, #database, #typeorm